### ISSUE TITLE
FR: Add new secondary info option - logbook_last_changed

### ISSUE THREAD

**Issue by danielbrunt57** [benct/lovelace-multiple-entity-row#257]
...or something like that.
My idea stems from this [custom logbook card](https://github.com/royto/logbook-card) which can display an entity's history from the logbook.  I am only interested in the most recent logbook entry rather than being stuck with HA's `secondary_info:` fields `last_updated` & `last_changed` which all get reset when HA starts/restarts.

![image](https://user-images.githubusercontent.com/49846893/180667639-e722e172-feae-428a-a9e6-a463f37c2933.png)

My HA was restarted 2 hours ago and so `last_updated` & `last_changed` are all now 2 hours ago...

![image](https://user-images.githubusercontent.com/49846893/180667732-ef15be77-60ea-47b0-b49f-4f868d24500d.png)

**Comment by ildar170975** [comment]
> My HA was restarted 2 hours ago and so `last-updated` & `last-changed` are all now 2 hours ago...

Do you want a custom card to keep somewhere an actual `last-changed` value for ALL entities?

**Comment by danielbrunt57** [comment]
> Do you want a custom card to keep somewhere an actual `last-changed` value for ALL entities?
No, custom:logbook-card already does that by using data from recorder.

> A card may only display/process some existing states/attributes. Since an object does not keep these "actual last-changed/updated" - there is nothing to display/process.
I understand that.

> What if an entity is excluded from Recorder?
The onus is on the user to ensure history is recorded.  If there's none then programming would display `unavailable`

> Finally, I do not see any connection between `multiple-entity-row` and the Logbook.
There currently is no connection but if you could incorporate the logic that custom:logbook-card uses to access recorder then this proposed new attribute logbook-last-changed couold display the most recent entry from recorder, or `unavailailable` if the routine could not find the entity in recorder.

Ideally, HA should preserve the last-changed (preferred) or last-updated (not sure how or when this ever differs from last-changed) across restarts but since it currently does not despite 2+ years of people requesting it, I was hoping it could somehow be pulled from recorder the way that [custom:logbook-card](https://github.com/royto/logbook-card) does.  Using that card does not fulfill my desire to have the correct last-changed value in entity rows in other cards such as yours.  
I hope I am making myself clear.  If you do not feel it is not do-able or is too much work then so be it.  Thanks, Daniel.

**Comment by ildar170975** [comment]
> No, custom:logbook-card already does that by using data from recorder.

Are you sure that actual last-changed/updated are stored in DB?
Because AFAIK these data are NOT stored.

> display the most recent entry from recorder

Currently ALL cards display the latest values of objects in DB. And Recorder keeps a historical data for objects which are assigned to be recorded.

**Comment by danielbrunt57** [comment]
> > No, custom:logbook-card already does that by using data from recorder.
> 
> Are you sure that actual last-changed/updated are stored in DB? Because AFAIK these data are NOT stored.
> 
`custom:logbook-card` is NOT using HA's `last-changed`/`last-updated`. 
It is retrieving data from HA recorder/logbook.
In my posted image I am specifying `max-items: 1` to limit the list to just the newest entry.

![image](https://user-images.githubusercontent.com/49846893/180851350-043f952c-7255-46d9-8467-49321a97add3.png)

> > display the most recent entry from recorder
> 
> Currently ALL cards display the latest values of objects in DB. 
**>> And Recorder keeps a historical data for objects which are assigned to be recorded <<**
That is what I am looking for.

**Comment by ildar170975** [comment]
> It is retrieving data from HA recorder/logbook.

Recorder & Logbook are integrations using data from DB.
If some info is not present in DB - no card can use it.

> That is what I am looking for.

Do not want to seem rude but I do not understand you.
Once again.

First you asked about "real" last-changed/updated values (i.e. displayed like "12 minutes ago", stored as timestamps) - then I explained you that these data are not present in DB. The last-changed/updated values - as you noticed - are not persistent and do not survive HA restart.

If you need to display the latest data of some entity - a state or attributes - then ALL cards like Entities, Entity, multiple-entity-row DO DISPLAY the LATEST data. They are taken from DB.
Cards like history-graph take data from DB - if an entity is assigned for Recorder.
Logbook card take data from DB - if an entity is assigned for Recorder and this entity has no UoM specified.

Also, please use ">" with some caution 'cause sometimes you are formatting your answers like quotes.

**Comment by jpettitt** [comment]
Not planning this — logbook entries come from a separate history API rather than entity state, so the row would need its own subscription and cache to show something the logbook card already does well. Closing as not planned.
